src/modules/auth/component/Login/Login.tsx

```diff
@@ -76,41 +68,56 @@ export function Login() {
       >
         <Grid container spacing={2}>
           <Grid item sm={12}>
-
-            <form onSubmit={handleSubmit}>
+            <form onSubmit={handleSubmit(onSubmit)}>
               <Grid container spacing={6}>
                 <Grid item sm={12}>
-                  <TextField
-                    label="Email"
-                    placeholder="email@gmail.com"
+                  <Controller
```

Comment:
Create a new component called TextField and use inside of it MuiTextField.
Now, it should get the props from MuiTextFieldProps and also name, control, rules.
This way, it will be easy to integrate and use

Reply:
I already have such component
![image](https://github.com/user-attachments/assets/58a2bbb7-ad94-46b2-9072-04ca8b9c3c31)

Reply:
done